src/isomorphic/children/onlyChild.js

```diff
@@ -16,7 +16,7 @@ var invariant = require('fbjs/lib/invariant');
  * Returns the first child in a collection of children and verifies that there
  * is only one child in the collection.
  *
- * See https://facebook.github.io/react/docs/react-api.html#react.children.only
+ * See https://reactjs.org/docs/react-api.html#react.children.only
```

Comment:
A couple of these anchor tags are invalid b'c of how Gatsby handles them. For instance, this one should now be: reactjs.org/docs/react-api.html#reactchildrenonly

But this is not related to this PR 😄 Just mentioning it in passing.